sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Added `identifier_from_raw_id` and ensured that `CommunicationIdentifier.raw_id` is populated on creation. Together, these can be used to translate between a `CommunicationIdentifier` and its underlying canonical raw ID representation. Developers can now use the raw ID as an encoded format for identifiers to store in their databases or as stable keys in general.
 
 ### Breaking Changes
+- Added support for a new communication identifier `MicrosoftBotIdentifier`.
```

Comment:
should be it under the added features?

Reply:
I'd go with both...see my comment in the js sdk

Reply:
Updated